src/Core/src/Handlers/SearchBar/SearchBarHandler.iOS.cs

```diff
@@ -214,6 +214,10 @@ void OnCancelClicked(object? sender, EventArgs args)
 			void OnSearchButtonClicked(object? sender, EventArgs e)
 			{
 				VirtualView?.SearchButtonPressed();
+				if (Handler is SearchBarHandler handler)
```

Comment:
Better something like:
`(sender as UISearchBar)?.ResignFirstResponder();`

Reply:
> Better something like: `(sender as UISearchBar)?.ResignFirstResponder();`

Hi @jsuarezruiz ,
We have added the **(sender as SearchView)?.ClearFocus();** method after the search button tap action is completed. Could you please verify these changes?